### ISSUE TITLE
Support for shadowsocksr obfuscator

### ISSUE THREAD

**Issue by Indexyz** [blinksocks/blinksocks#95]
* **I'm submitting a...**

- [x] feature request

* **What is the current behavior?**
Now blinksocks is support some feature of shadowsocksr link `auth-chain-a`, but it don't support the shadowsocksr obfuscator right now. 

* **What is the expected behavior?**
Support shadowsocksr obfuscator like `tls1.2_ticket_auth` or `http_post`

* **What is the motivation / use case for changing the behavior?**
It make the project full support for shadowsocksr

**Comment by micooz** [comment]
Yep, I used to put it to my plan list, but I think those obfuscators are designed too disgusting. It's better to [implement](https://github.com/blinksocks/blinksocks/tree/master/docs/development/custom-preset) them as **thirdparty libraries** without putting them into blinksocks code base.
